src-tauri/src/state/user_prompt.rs

```diff
@@ -69,11 +76,11 @@ pub struct ShareCredentials {
 #[serde(untagged)]
 #[ts(export, export_to = "bindings/user-prompt/CurrentUserPrompt.ts")]
 pub enum CurrentUserPrompt {
+    PasswordRequired(PasswordRequired),
```

Comment:
Minor issue: If I recall correctly moving this up was initially in order to temporarily solve the weird deserialization issue. Since that has been resolved now with the help of `#[serde(deny_unknown_fields)]` `PasswordRequired(PasswordRequired),` can be placed back to it's original position below `    CredentialOffer(CredentialOffer),`.

In case if there's another reason for changing the order in this enum, then please also commit the corresponding change that `ts_rs` will generate in `bindings/user-prompt/CurrentUserPrompt.ts` when you execute `cargo test` in `src-tauri`.